packages/transitive-overlay/src/route-layers.ts

```diff
@@ -32,16 +39,58 @@ export function patternToRouteFeature(
     .replace(/\s/g, "!")
     .replace(/[+-0124679FHJLPTVXYZ]/g, "E");
 
+  const properties = {
+    color: `#${route.route_color || "000080"}`,
+    name: routeName,
+    nameUpper: routeNameUpper,
+    routeType: route.route_type,
+    textColor: `#${route.route_text_color || "eee"}`,
+    type: "route"
+  };
+
+  const isFlex = pattern.stops[pattern.stops.length - 1].stop_id.includes(
+    "flexed_to"
+  );
+  const straight = polyline.toGeoJSON(polyline.encode(concatenatedLines));
+
+  if (isFlex) {
```

Comment:
(non-blocking) I think TNC itineraries used to be drawn using arcs too. Could you add a TNC condition (at the right spot) too? Or create an issue specific to TNCs.

Reply:
Have added a comment and an issue